README.md

```diff
@@ -1,5 +1,40 @@
 # GO meets Ansa
 
+## Dependencies
+
+Package github.com/julienschmidt/httprouter is used for request routing.
+
+`go get https://github.com/julienschmidt/httprouter`
+
+## Run application
+
+`go run cmd/server/main.go`
+
+And then make a request to http://localhost:8080/channels/Homepage, where Homepage is an example topic. 
+You should be able to get an xml response corresponding to the RSS channel.
+
+## Documentation
+
+`godoc -http=:6060`
+
+Go to http://localhost:6060/pkg/github.com/Owanesh/go-ansa/
```

Comment:
just as a reminder, you could just link the user to https://godoc.org/github.com/Owanesh/go-ansa,  since It'll be the same docs and the user doesnt have to handle a godoc server

Reply:
Thank you so much @arxdsilva and @nachogoca <3